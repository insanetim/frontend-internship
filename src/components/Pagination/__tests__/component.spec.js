import { shallow } from 'enzyme'

import Pagination from '../component'

it('matches snapshot', () => {
  const component = shallow(<Pagination />)

  expect(component).toMatchSnapshot()
})
