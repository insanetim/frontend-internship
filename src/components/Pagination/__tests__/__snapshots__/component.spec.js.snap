// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Row
  justify="center"
>
  <Col>
    <Pagination
      className="pagination"
      showSizeChanger={false}
    />
  </Col>
</Row>
`;
